### ISSUE TITLE
[req]: leech24

### ISSUE THREAD

**Issue by ClandestineClout** [Jackett/Jackett#12553]
### Is there already a request for your tracker?

- [X] I have checked older issues, open and closed

### Type

Private

### Tracker details

```markdown
- Name: Leech24
- URL: https://leech24.net/
- Description: leech24 is a Private Torrent Tracker for MOVIES / TV / GENERAL
- Minimum Seeding Time and Minimum Ratio (if Private): TBD
- Additional Comments: Open reg
no rules or FAQ yet
```
Engine: TT3.0

**Comment by garfield69** [comment]
v0.20.10

**Comment by ilike2burnthing** [comment]
https://leech24.net/ - ERR_CONNECTION_TIMED_OUT

**Comment by garfield69** [comment]
online, albeit with a CA issued to another domain

**Comment by ilike2burnthing** [comment]
https://leech24.net/ - Error 522 - Connection timed out

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
http 500

**Comment by ilike2burnthing** [comment]
> https://leech24.net/ - Error 522 - Connection timed out

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
I'm still getting HTTP ERROR 500 with the browser so I cannot fetch the cookie to use with the indexer :-(

**Comment by ilike2burnthing** [comment]
Given how up and down it has been over the past 10 days I wouldn't be surprised if it continues to be temperamental minute to minute. It's currently up for me and all Uptrends nodes show it as online too.

**Comment by garfield69** [comment]
site presenting 
![image](https://user-images.githubusercontent.com/6933240/159630706-a6964f9b-c759-484b-b60d-690c96d70ea5.png)
site looks like its gone

**Comment by ilike2burnthing** [comment]
Ah, they were running the site on a QNAP NAS, seems like a bad idea. Something has gone wrong and it's redirecting to the QNAP login page.

**Comment by garfield69** [comment]
With hide.me and a browser I was able to login, get my cookie and the Jackett indexer passed the test.
marking this as online.

**Comment by garfield69** [comment]
https://leech24.net/ DATABASE: mysqli_connect:

**Comment by ilike2burnthing** [comment]
back up, but with invalid cert

**Comment by garfield69** [comment]
v0.20.3035

**Comment by garfield69** [comment]
https://leech24.net/ DATABASE: mysqli_connect:

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
http 500

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
not for me. is this another windows or chromium crap out?

**Comment by garfield69** [comment]
yeah, got it to work by using proton vpn with a  Netherlands server.

**Comment by garfield69** [comment]
https://leech24.net/ ERR_CONNECTION_TIMED_OUT
all uptime checkpoints return TCP Connection Failed

**Comment by garfield69** [comment]
https://leech24.net/ HTTP ERROR 500 Internal Server Error

**Comment by ilike2burnthing** [comment]
Working ok here. Temporary issue?

**Comment by garfield69** [comment]
Not for me. 
Both chrome and firefox report error.
Ah. But when I tried proton it works.
So a local region issue.

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/12553#issuecomment-1660945013

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/12553#issuecomment-1660945013

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/12553#issuecomment-1660945013

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://leech24.net/ DNS Lookup Error   or  ERR_SSL_VERSION_OR_CIPHER_MISMATCH

**Comment by garfield69** [comment]
torrent trader -> unit3d 8.1.3

**Comment by garfield69** [comment]
my credentials not recognised, recovery email did not get sent, and registration is invite only ATM

**Comment by garfield69** [comment]
https://leech24.net/ ERR_CONNECTION_REFUSED

**Comment by garfield69** [comment]
Oh so close to working. 
Login page online, but my creds were not recognised. So I filled in the registration page, but before getting to the completed stage it gave out a 500 internal server error :-(

**Comment by garfield69** [comment]
https://leech24.net/ ERR_CONNECTION_REFUSED

**Comment by garfield69** [comment]
Back online, creds not recognised. signed up again, this time got to the part where you click to have the confirmation sent.
but not email yet. maybe I should have used proton mail ...

**Comment by ilike2burnthing** [comment]
Can now reset password, their email is working again.

**Comment by garfield69** [comment]
v0.22.217

**Comment by garfield69** [comment]
https://leech24.net/ HTTP ERROR 500 (Internal Server Error)

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/12553#issuecomment-2295486929

**Comment by garfield69** [comment]
Online, but my credentials are gone.
Registered anew, but had trouble on my chrome browser with the verification links which seem to be redirecting to godaddy tracking with invalid CA %-/
In the end did a pswd reset on firefox and managed to login.

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/12553#issuecomment-2295486929

**Comment by ilike2burnthing** [comment]
Tried activating and resetting, no luck.

**Comment by garfield69** [comment]
Finally, after nearly a month of daily trying to get the account verified the password reset email arrived!
The site has clearly been reset and is missing everything, forum topics, Rules, FAQ, wiki.
Accessing torrents either via WEB or API generates Error 500: Internal Server Error Our server encountered an internal error. Sorry For The Inconvenience
So we continue to wait ...

**Comment by garfield69** [comment]
v0.22.1323

**Comment by garfield69** [comment]
back to tt3.0
v0.22.1570

**Comment by ilike2burnthing** [comment]
Site up but https://leech24.net/torrents.php shows zero torrents.

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://leech24.net/torrents-search.php?search=&cat=0&incldead=1&freeleech=0&lang=0 > no results

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
did you check the indexer or just checked the web site?
I still get no results.
Will check if the indexer needs updating.

**Comment by garfield69** [comment]
v0.24.790

**Comment by ilike2burnthing** [comment]
Huh, weird. Yea, site and indexer were both working for me, but that change works too :P

**Comment by garfield69** [comment]
https://leech24.net/ > TCP Connection Failed (timed out)

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
Power to house out since yesterday and estimate is 2 more days of outage.
Computer down so I use this emg way to let you know I'm not online. See you
at the other end ;-) garfield69
